src/index.js

```diff
@@ -0,0 +1,28 @@
+import apis from '../apis.json';
```

Comment:
Ah maybe that was a browserify problem. Webpack uses a json-loader to handle this.